.travis.yml

```diff
@@ -1,26 +1,29 @@
 ---
 language: python
 
-python:
-    - 3.6
-    - 3.7
-
 env:
-    - CTAPIPE_VERSION="v0.7.0" CTAPIPE_IO_LST_VERSION="v0.4.1" PROTOZFITS_VERSION=v1.4.2
```

Comment:
I have here the same comment as in https://github.com/cta-observatory/ctapipe_io_lst/pull/32, could you add the ctapipe v0.8 test as well?